devito/types/dense.py

```diff
@@ -1448,7 +1456,7 @@ def __shape_setup__(cls, **kwargs):
 
     @cached_property
     def _fd_priority(self):
-        return 2.1 if self.staggered in [NODE, None] else 2.2
+        return 2.1 if not self.staggered or all(s == 0 for s in self.staggered) else 2.2
```

Comment:
makes me think `staggered` should be of type `class Staggering(tuple)` rather just a tuple